torch/csrc/autograd/variable.cpp

```diff
@@ -298,12 +356,14 @@ Tensor VariableHooks::tensor_data(const Tensor& self) const {
   return at::Tensor(self_impl_copy);
 }
 
-// View Variables
+// Backward View Variables
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 bool VariableHooks::is_view(const Tensor& self) const {
-  if (torch::autograd::impl::get_autograd_meta(self)) {
-    return torch::autograd::impl::get_autograd_meta(self)->is_view_;
+  auto meta = torch::autograd::impl::get_autograd_meta(self);
+  if (meta && meta->is_view_) {
+    auto diff_view_meta = static_cast<torch::autograd::DifferentiableViewMeta*>(torch::autograd::impl::get_autograd_meta(self));
+    return diff_view_meta->has_bw_view();
```

Comment:
Wondering if there shouldn't have been some virtual methods here instead of having to do static casting, but apparently this is the prevailing style so we can let it go